tests/test_models.py

```diff
@@ -92,6 +92,9 @@ class _ModelTestCase:
     #: Additional arguments passed to the model's constructor method
     model_kwargs: ClassVar[Optional[Mapping[str, Any]]] = None
 
+    #: Additional arguments passed to the training loop's constructor method
+    training_loop_kwargs: ClassVar[Optional[Mapping[str, Any]]] = None
```

Comment:
also evaluator kwargs?

Reply:
It seems that we don't use the evaluator in these tests.